apps/molgenis-components/src/components/forms/InputInt.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <FormGroup :id="id" :label="label" :description="description">
+    <input
+      type="number"
+      step="1"
+      :value="value"
+      class="form-control"
+      :aria-describedby="id + 'Help'"
+      :placeholder="placeholder"
+      @keypress="handleKeyValidity"
+      @input="emitIfValid"
+    />
+  </FormGroup>
+</template>
+
+<script>
+import BaseInput from "./BaseInput.vue";
+import FormGroup from "./FormGroup.vue";
+import { isNumericKey } from "./utils/InputUtils";
+
+export default {
+  extends: BaseInput,
+  components: {
+    FormGroup,
+  },
+  methods: {
+    emitIfValid(event) {
+      const value = parseInt(event.target.value);
+      if (event.target.value === "") {
+        this.$emit("input", null);
+      }
+      if (!isNaN(value)) {
+        this.$emit("input", value);
+      }
+    },
+    handleKeyValidity(event) {
```

Comment:
I realize this is a new feature request so you can ignore. However, LongInput has validation on wether it is a big int. Int doesn't. I propose we more or less merge LongInput and IntInput with only difference the min and max?

Reply:
One big difference is that Long is actually a string while int returns a number. So if we want to merge them we'd have to think about how to treat them. InputInt is a number in order to use it in expressions.

Reply:
ah, then ignore my 'merge' comment.
My only comment remaining is that the 'Int' will not check if it goes beyond what is allowed. Server will complain though. So you decide if you want to make the change or postpone for later.